libraries/bxdf/standard_surface.mtlx

```diff
@@ -19,33 +19,33 @@
+      <input name="specular_IOR" type="float" value="1.5" uimin="0.0" uisoftmax="3.0" uiname="Index of Refraction" uifolder="Specular"
             doc="Index of refraction for specular reflection." />
       <input name="specular_anisotropy" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Specular Anisotropy" uifolder="Specular" uiadvanced="true"
             doc="The directional bias of reflected and transmitted light resulting in materials appearing rougher or glossier in certain directions." />
-      <input name="specular_rotation" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Specular Rotation" uifolder="Specular" uiadvanced="true"
+      <input name="specular_rotation" type="float" value="0" uisoftmin="0.0" uisoftmax="1.0" uiname="Specular Rotation" uifolder="Specular" uiadvanced="true"
             doc="Rotation of the axis of specular anisotropy around the surface normal." />
       <input name="transmission" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Transmission" uifolder="Transmission" uiadvanced="true"
             doc="Transmission of light through the surface for materials such as glass or water. The greater the value the more transparent the material." />
       <input name="transmission_color" type="color3" value="1, 1, 1" uimin="0,0,0" uimax="1,1,1" uiname="Transmission Color" uifolder="Transmission" uiadvanced="true"
             doc="Color tint on the transmitted light." />
-      <input name="transmission_depth" type="float" value="0" uimin="0.0" uimax="100.0" uiname="Transmission Depth" uifolder="Transmission" uiadvanced="true"
+      <input name="transmission_depth" type="float" value="0" uimin="0.0" uisoftmax="100.0" uiname="Transmission Depth" uifolder="Transmission" uiadvanced="true"
             doc="Specifies the distance light travels inside the material before its becomes exactly the transmission color according to Beer's law." />
       <input name="transmission_scatter" type="color3" value="0, 0, 0" uimin="0,0,0" uimax="1,1,1" uiname="Transmission Scatter" uifolder="Transmission" uiadvanced="true"
             doc="Scattering coefficient of the interior medium. Suitable for a large body of liquid or one that is fairly thick, such as an ocean, honey, ice, or frosted glass." />
       <input name="transmission_scatter_anisotropy" type="float" value="0" uimin="0.0" uimax="1.0" uiname="Transmission Anisotropy" uifolder="Transmission" uiadvanced="true"
             doc="The amount of directional bias, or anisotropy, of the scattering." />
-      <input name="transmission_dispersion" type="float" value="0" uimin="0.0" uimax="100.0" uiname="Transmission Dispersion" uifolder="Transmission" uiadvanced="true"
+      <input name="transmission_dispersion" type="float" value="0" uimin="0.0" uisoftmax="100.0" uiname="Transmission Dispersion" uifolder="Transmission" uiadvanced="true"
```

Comment:
@niklasharrysson , do we want a soft max for this as it's a physical property and 100 means something.

Reply:
Yes, soft max of 100 sounds good for this. It's the Dispersion Abbe number, which if enabled is normally in the range 10-70. It actually gives a higher dispersion effect the lower the number but at 0 it's disabled. So 0-100 is good, where 100 is a soft max.